bundles/org.eclipse.orion.client.ui/web/orion/parameterCollectors.js

```diff
@@ -81,8 +81,8 @@ define(['i18n!orion/nls/messages', 'orion/webui/littlelib'],
 				this._activeElements.onClose = onClose;
 				var focusNode = fillFunction(this._activeElements.parameterArea, this._activeElements.dismissArea);
 				if (!focusNode) {
-					// no parameters were generated.  
-					return false;
+					// no required parameters
+					return true;
```

Comment:
We would ideally not have to rely on the return value to tell us when there are no required parameters, but unfortunately none of the three parameters to this function, commandNode, fillFunction, or onClose has that information. Changing the method signature makes merge conflicts worse and forces us to update all other places in the codebase where it is used.